app/experimenter/nimbus-ui/src/components/PageResults/TableMetricCount/MetricHeader.tsx

```diff
@@ -0,0 +1,67 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React, { useContext } from "react";
+import { ReactComponent as Info } from "../../../images/info.svg";
+import { ResultsContext } from "../../../lib/contexts";
+import { METRIC_TYPE } from "../../../lib/visualization/constants";
+import TooltipWithMarkdown from "../TooltipWithMarkdown";
+
+type MetricTypes =
+  | typeof METRIC_TYPE.PRIMARY
+  | typeof METRIC_TYPE.USER_SELECTED_SECONDARY
+  | typeof METRIC_TYPE.DEFAULT_SECONDARY
+  | typeof METRIC_TYPE.GUARDRAIL;
+
+type MetricHeaderProps = {
+  outcomeSlug: string;
+  outcomeDefaultName: string;
+  metricType?: MetricTypes;
+};
+
+const MetricHeader = ({
+  outcomeSlug,
+  outcomeDefaultName,
+  metricType = METRIC_TYPE.DEFAULT_SECONDARY,
+}: MetricHeaderProps) => {
+  const {
+    analysis: { metadata },
+  } = useContext(ResultsContext);
+
+  const outcomeName =
+    metadata?.metrics[outcomeSlug]?.friendly_name || outcomeDefaultName;
+  const outcomeDescription =
+    metadata?.metrics[outcomeSlug]?.description || undefined;
```

Comment:
Why `undefined` here and not `null`? Just curious.

Reply:
As you noticed, I just refactored existing code and this moved along, but the important thing is that it's falsy if there is no `description` value. Personally I think `undefined` makes sense here, but either would work.